test/pylib/cpp/boost/boost_facade.py

```diff
@@ -113,6 +119,12 @@ def read_file(name: Path) -> str:
         if original_name != Path(executable).stem:
             if executable.stem == COMBINED_TESTS.stem:
                 args.append(f"--run_test={file_name.stem}/{original_name}")
+            # The executables that have an executable but also have an entry in self.combined_suites are
+            # “separated executables” that were explicitly built by the developer.
```

Comment:
This will not work when combined test binary is absent. This is the whole idea of this PR to not build combined tests. Please rewrite `list_tests` method in `BoostTestFacade` to support optional suits.